client/src/pages/About.jsx

```diff
@@ -1,6 +1,8 @@
 const About = () => (
 	<>
-		<h2>About</h2>
+		<h2>
+			<span>About</span>
+		</h2>
```

Comment:
Great approach! Thanks for your efforts! To keep our workflow streamlined and ensure clarity, it would be even better to create separate PRs for changes that aren’t directly related to the feature. This makes it easier to review and track changes independently.